src/Angor/Avalonia/AngorApp.Desktop/app.manifest

```diff
@@ -3,7 +3,7 @@
   <!-- This manifest is used on Windows only.
        Don't remove it as it might cause problems with window transparency and embedded controls.
        For more details visit https://learn.microsoft.com/en-us/windows/win32/sbscs/application-manifests -->
-  <assemblyIdentity version="1.0.0.0" name="AngorApp.Desktop"/>
+  <assemblyIdentity version="1.0.0.0" name="Angor"/>
```

Comment:
preparation for mobile later down the line?

Reply:
In this case it was to make the application icon in Linux show "Angor" instead of "Angor.Desktop". This made the trick. Anyhow, I hope mobile is working as well as desktop in the near future. I haven't tried how it goes on Android yet, but I will do a bit tests, probably next week.

Reply:
Yeah brilliant, would be good to know.